.golangci.yml

```diff
@@ -0,0 +1,44 @@
+run:
```

Comment:
The change of this file is in the commit a7b467feb8e71ad2978632580e529dec0b7ab4f1. Is this what you expected?
I thougt that using kubebuilder go/v4 template and enabling lint should be in the different commit.

Reply:
Commit splits have been revised. There are two commits related to lint.
https://github.com/topolvm/pvc-autoresizer/pull/236/commits/e9077649d9ee8cb4c343ce1b64d0a139ca282075
https://github.com/topolvm/pvc-autoresizer/pull/236/commits/0caf6cf1bcf7d6bd15a4947ff679171c6c1fc324